go/test/endtoend/vreplication/cluster_test.go

```diff
@@ -87,11 +89,29 @@ type ClusterConfig struct {
 	vreplicationCompressGTID bool
 }
 
+func (cc *ClusterConfig) compressGTID() func() {
+	cc.vreplicationCompressGTID = true
+	return func() {
+		cc.vreplicationCompressGTID = false
+	}
+}
```

Comment:
IMO we should change the name as it confused me at first: `enableCompressGTIDs()` And a comment would be good.

Reply:
done